airflow/models.py

```diff
@@ -106,6 +106,10 @@ def clear_task_instances(tis, session, activate_dag_runs=True):
             if ti.job_id:
                 ti.state = State.SHUTDOWN
                 job_ids.append(ti.job_id)
+        # todo: this creates an issue with the webui tests
```

Comment:
Future. For lineage purposes this needs to be done, now you would loose that. verify_integrity will recreate the tasks if required.